### PR TITLE
[MACAWSUP-53] Reflect email branding

### PR DESCRIPTION
**Pull request by steven-chang-maestrano**
Macaw wanted branded email subjects (along with other aspects of the email).

### CODE REVIEW

**Comment by steven-chang-maestrano** [comment]
@ouranos:
- Changed based to 4.0
- specs written but couldn't get it to run on rspec, I get this error: 
Do I need to do something different?

screen shot 2019-02-04 at 2 55 00 pm

**Comment by ouranos** [comment]
@steven-chang-maestrano build failed. To see how to run the specs have a look at `.travis.yml` or at https://github.com/maestrano/mno-enterprise/blob/4.0/docs/testing.md

**Comment by steven-chang-maestrano** [comment]
@ouranos - Tests have passed and changed base to 4.0. Please review and merge when possible. Let me know if any issues. Thanks

File: core/spec/lib/mno_enterprise/smtp_client_spec.rb [hunk 1]
```diff
@@ -2,54 +2,93 @@
 
 describe MnoEnterprise::SmtpClient do
   let(:client) { described_class.send(:new) }
+  let(:from_object) { { name: 'John', email: 'john.doe@example.com' } }
```
Comment (ouranos) [inline core/spec/lib/mno_enterprise/smtp_client_spec.rb]: I'm not sure you need to extract all params to their own variable (also `from_object` is not the best name ;) )

```ruby
let(:opts) { nil }
let(:params) do
  [
    'reset-password-instructions',
    {name: 'John', email: 'john.doe@example.com'},	
    {name: 'Joe', email: 'joe.blogg@example.com'},
    attachments: [	
      {	
        name: 'some-file.pdf',	 
        content: 'the file'
      }
    ],
   opts	 
  ] 
end

subject { client.deliver(*params) }
```

Then your context can be:
```ruby
context 'with a specified subject' do
  let(:opts) { subject: 'user specified subject' }
  it 'uses the specified subject' do
    expected_params[:subject] = user_specified_subject
    ...
  end
end
```

You don't need to explicitly test `it 'sets the subject of the email to the #humanize(template)'` as it's tested by `calls mail with the correct params`

Then you don't need to add all the extra context as it's the default ('opts has not been inputted by user').
Also note the context wording as recommended by betterspecs (with/when).

**Comment by cesar-tonnoir** [comment]
@ouranos I've applied your recommendations and merged with #803 
Can you please tag when you have a chance? Or do you want me to do it?
I'll lock macaw-enterprise to the commit number in the meantime

**Pull request merged**